generators/server/index.js

```diff
@@ -434,16 +445,16 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
     return {
```

Comment:
why not moving this one in server/cleanup.js?

Reply:
With modularization, cleanup will be small. I don't see a need for a separated file.